pkg/agent/qrm-plugins/memory/memory.go

```diff
@@ -17,10 +17,29 @@ limitations under the License.
 package memory
 
 import (
+	"fmt"
+	"time"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+
 	"github.com/kubewharf/katalyst-core/cmd/katalyst-agent/app/agent/qrm"
 	"github.com/kubewharf/katalyst-core/pkg/agent/qrm-plugins/memory/dynamicpolicy"
+	"github.com/kubewharf/katalyst-core/pkg/agent/qrm-plugins/memory/handlers/sockmem"
+	"github.com/kubewharf/katalyst-core/pkg/agent/utilcomponent/periodicalhandler"
 )
 
 func init() {
 	qrm.RegisterMemoryPolicyInitializer(dynamicpolicy.MemoryResourcePluginPolicyNameDynamic, dynamicpolicy.NewDynamicPolicy)
 }
+
+// register qrm memory-handlers registered in adapter
+func init() {
+	var errList []error
+	errList = append(errList, periodicalhandler.RegisterPeriodicalHandler(qrm.QRMMemoryPluginPeriodicalHandlerGroupName,
```

Comment:
will it be better for this strategy applied through `cgroup manager` or `cri` cc @csfldf

Reply:
@csfldf Any idea?

Reply:
can we move the registration code to [DynamicPolicy.Start](https://github.com/kubewharf/katalyst-core/blob/main/pkg/agent/qrm-plugins/memory/dynamicpolicy/policy.go#L243), and refine it like below: 

if p.enableXxx {
   err := periodicalhandler.RegisterPeriodicalHandler(...)

   if err != nil {
        return fmt.Error("xxx failed with error: %v", err);
   }
}

Reply:
Done.